docs/RFCS/monitoring_queries_and_sessions.md

```diff
@@ -0,0 +1,219 @@
+| node_id | username | client_address   | application_name | active_queries                                       | session_start | oldest_query_start | kv_txn_id |
++---------+----------+------------------+------------------+------------------------------------------------------+---------------+--------------------+-----------+
+| 1       |  root    | 192.168.1.5:8080 | test             | SELECT * FROM users; INSERT INTO test VALUES (...);  | 1234567890    | 1234567891         | 1e5f6bac  |
+| 2       |  bilal   | 192.168.1.7:8080 | pg.js            |                                                      | 1494524202    |                    |           |
++---------+----------+------------------+------------------+------------------------------------------------------+---------------+--------------------+-----------+
+(2 rows)
+
+```
+
+Note that the `active_queries` field is a delimeter-separated list of all queries currently
+executing under that session, with SQL parallelization taken into account.
+
+Additions to the SQL grammer will be made to support filtering using SELECT statements and
+nesting of SHOWs as tables:
+
+```
+
+root@:26257/> SELECT * FROM (SHOW SESSIONS) AS a WHERE a.username = 'bilal';
+
++---------+----------+------------------+------------------+------------------------------------------------------+---------------+--------------------+-----------+
+| node_id | username | client_address   | application_name | active_queries                                       | session_start | oldest_query_start | kv_txn_id |
++---------+----------+------------------+------------------+------------------------------------------------------+---------------+--------------------+-----------+
+| 2       |  bilal   | 192.168.1.7:8080 | pg.js            |                                                      | 1494524202    |                    |           |
++---------+----------+------------------+------------------+------------------------------------------------------+---------------+--------------------+-----------+
+(1 row)
+
+```
+
+Similarly, `SHOW REQUESTS [GLOBAL|LOCAL]` will return currently executing queries.
+
+Example usage:
+
+```
+
+root@:26257/> SHOW REQUESTS
+
++---------+----------+------------------+------------------+--------------------------------+-------------+
+| node_id | username | client_address   | application_name | query                          | start       |
++---------+----------+------------------+------------------+--------------------------------+-------------+
+| 1       |  root    | 192.168.1.5:8080 | test             | SELECT * FROM users;           | 1234567894  |
+| 1       |  root    | 192.168.1.5:8080 | test             | INSERT INTO test VALUES (...); | 1234567891  |
++---------+----------+------------------+------------------+--------------------------------+-------------+
```

Comment:
I think you should move ip and applicastion_name to the end. start time can be age as a duration.